test/test_regression_cli.py

```diff
@@ -94,8 +116,18 @@ def check_postgres_regression(self, dataset, known_md5):
         current_md5 = getmd5(unixfileformat("output_file")) 
         assert current_md5 == known_md5
 
+
+class DownloadRegression(TestCase):
+    def check_download_regression(self, dataset, known_md5):
+        """Check for regression for a particular dataset downloaded only"""
+        os.system("retriever download %s -p raw_data/%s" %(dataset,dataset))
```

Comment:
We should use the newer `.format` instead of `%` to help set us up for the shift to Python 3 at some point. This line would become:

`"retriever download {0} -p raw_data/{0}".format(dataset)`

The indexing here (i.e., the use of `{0}` instead of `{}`) lets us reuse the first value inside `.format()` instead of listing it twice. Make a similar change to the next line.